### ISSUE TITLE
Supported web technologies for Webview

### ISSUE THREAD

**Issue by Laegel** [tauri-apps/tauri-docs#33]
Nice suggestion coming from Discord:

> Perhaps someone could create a Union of the browsers that are represented by Webview? Something like: https://caniuse.com/#compare=ie+10,ie+11,edge+80,firefox+74,chrome+80

Requires investigation, definitely an important information.

**Comment by Laegel** [comment]
http://html5test.com/
https://developer.mozilla.org/en-US/docs/Web/API/CSS/supports
https://developer.mozilla.org/fr/docs/Web/CSS/@supports

**Comment by simonhyll** [comment]
I don't think we can make a full on "caniuse" type of page for this in the documentation, but we can certainly add information about how to figure this sort of thing out in the docs. Tracking for 2.0 so we don't forget to bring it up somewhere.
